Automattic/jetpack
# Admin menu: Move site editor menu item and add beta badge

Fixes #

#### Changes proposed in this Pull Request:

* Right now we remove the site editor menu and re-add it without the beta badge. In this PR we remove all that logic and keep the original site editor menu. We make sure to replace the link for Calypso though.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* See https://github.com/Automattic/wp-calypso/pull/54681

Related to https://github.com/Automattic/wp-calypso/issues/54464
